wordpress-mobile/WordPress-iOS
# Add Google framework for login

Refs #7675 

To test:
- acquire a copy of `GoogleKeys.xcconfig` that contains valid Google ID
- ensure the wordspace still builds
- tap the "Login in with Google" button
- login to Google
- ensure the success alert appears

*Discussion:*
To manage the Google API that requires an `Info.plist` config, the API keys are stored in a new xcconfig file that will not be added to the repo. This will require a new file setup from developers, and modifications to documentation.

Needs review: @aerych
